<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="createDeckButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>80</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="text">
     <string>Create Deck</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteDeckButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>140</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Deck</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filterDeckButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>260</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter Deck</string>
    </property>
   </widget>
   <widget class="QPushButton" name="statsButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>320</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Check Stats</string>
    </property>
   </widget>
   <widget class="QLabel" name="appTitle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Flashcard Study App</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Deck1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>Deck</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Deck2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>Deck</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Deck3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>Deck</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Deck4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>Deck</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Deck6">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>240</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>Deck</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Deck5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>Deck</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Deck7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>Deck</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Deck9">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>390</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>Deck</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Deck8">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>390</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>Deck</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextPageButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>550</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prevPageButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>550</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous</string>
    </property>
   </widget>
   <widget class="QLabel" name="pageLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>550</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Page 1 of 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="renameDeckButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>200</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Rename Deck</string>
    </property>
   </widget>
   <widget class="QPushButton" name="importDeckButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>380</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Deck</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportDeckButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>440</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Export Deck</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportAllButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>500</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Export All Decks</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
